phantomjs/main.js

```diff
@@ -142,5 +142,15 @@ page.onLoadFinished = function(status) {
   }
 };
 
+page.onResourceError = function(resourceError)
+{
+  sendMessage('error.resourceError', resourceError);
+};
+
+page.onResourceTimeout = function(request)
+{
+  sendMessage('error.onResourceTimeout', request);
+};
+
```

Comment:
The `{` and `function(...)` should be on the same line.

Also, I think you have to change `error.resourceError` to `error.onResourceError` to match the convention used for other events handlers.